apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -288,6 +321,8 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
 
     const property = useMemo(() => organizationProperties.find(property => property.id === selectedPropertyId), [organizationProperties, selectedPropertyId])
 
+    const isExistedTicket = get(initialValues, 'property')
```

Comment:
is it a hack?

Reply:
Moved this variable to an external component